main.go

```diff
@@ -1,48 +1,132 @@
 
 <p>Id enim volumus, id contendimus, ut officii fructus sit ipsum officium. Quo modo? Expressa vero in iis aetatibus, quae iam confirmatae sunt. Quodsi vultum tibi, si incessum fingeres, quo gravior viderere, non esses tui similis; Esse enim, nisi eris, non potes. </p>`
 
-	lines := strings.Split(lorem, "\n")
+func print_frequency_map(frequency_map map[int][]string) {
+	for key, element := range frequency_map {
+		fmt.Println(key)
+		for _, word := range element {
+			fmt.Println(word)
+		}
+	}
+}
+
+type WordCount struct {
+	Word  string `json:"Word"`
+	Count int    `json:"Count"`
+}
+
+func word_json(lorem io.Reader) string {
+	// url := "loripsum.net/api"
+
+	newScanner := bufio.NewScanner(lorem)
 
-	// fmt.Println(lines)
-	// fmt.Println(len(lines))
+	newScanner.Split(bufio.ScanWords)
 
 	counts := map[string]int{}
 
-	for _, line := range lines {
-		words := strings.Split(line, " ")
-		for _, word := range words {
-			counts[word]++
-		}
+	for newScanner.Scan() {
+		counts[newScanner.Text()]++
+	}
+
+	var wordCountArray []WordCount
+	for key, value := range counts {
+		wordCountArray = append(wordCountArray, WordCount{key, value})
 	}
 
-	j, err := json.Marshal(counts)
+	sort.Slice(wordCountArray, func(p, q int) bool {
+		if wordCountArray[p].Count != wordCountArray[q].Count {
+			return wordCountArray[p].Count > wordCountArray[q].Count
+		}
+		return wordCountArray[p].Word > wordCountArray[q].Word
+	})
+
+	j, err := json.Marshal(wordCountArray)
 	if err != nil {
 		panic(err)
 	}
+	return string(j)
+
+	/*var s []string
+
+	frequency_map := map[int][]string{}
+
+	frequencies := map[int]bool{}
 
-	fmt.Println(string(j))
+	for key, element := range counts {
+		frequency_map[element] = append(frequency_map[element], key)
+		frequencies[element] = true
+	}
+
+	frequency_array := make([]int, 0, len(frequencies))
+
+	for key := range frequencies {
+		frequency_array = append(frequency_array, key)
+	}
+
+	sort.Slice(frequency_array, func(p, q int) bool { // sorts by value in array, ascending to descending
+		return frequency_array[p] > frequency_array[q]
+	})
+
+	little_map := map[string]int{}
+
+	for _, key := range frequency_array {
+		fmt.Println("zoom")
+		fmt.Println(frequency_array[key])
+		for _, element := range frequency_map[key] {
+			little_map[element] = key
+			j, err := json.Marshal(little_map)
+			if err != nil {
+				panic(err)
+			}
+			delete(little_map, element)
+			s = append(s, string(j)+",")
+		}
+	}
+	s[len(s)-1] = s[len(s)-1][:len(s[len(s)-1])-1]
+	return s*/
+}
+
+func main() {
+
+	http.HandleFunc("/", indexPage)
+	http.ListenAndServe(":8000", nil)
+
+}
+
+func indexPage(w http.ResponseWriter, r *http.Request) {
+
+	switch r.Method {
+	case http.MethodGet:
+		myReader := strings.NewReader(lorem)
+		s := word_json(myReader)
+		fmt.Fprintf(w, s)
```

Comment:
In general, having two `case` blocks with the same content sticks out as a code smell to me. Here, you can probably get rid of this specific case and let the `default` case handle it.

If I were being an HTTP pedant, I'd suggest keeping this block the same and returning an HTTP 405 "Method Not Allowed" error in the `default` case.